src/transport/raw/TCP.h

```diff
@@ -187,7 +187,7 @@ class DLL_EXPORT TCPBase : public Base
     // and release from the pool.
     void TCPDisconnect(Transport::ActiveTCPConnectionState * conn, bool shouldAbort = false) override;
 
-    bool IsServerListenEnabled() override;
+    bool IsServerListenEnabled();
```

Comment:
Is this used?  It looks unused....

Reply:
Not used currently. Kept it thinking it might be of use in the future and it is a single boolean function. There are 2 unit tests that I added for this and removing this would require removing them as well.

Reply:
Removed this one. Modified the test case to check the TcpListenParameters variable.